<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraPositionView</class>
 <widget class="QWidget" name="CameraPositionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>107</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignHCenter">
    <widget class="PMDButton" name="frontViewButton">
     <property name="text">
      <string>Front View</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="PMDButton" name="sideViewButton">
     <property name="text">
      <string>Side View</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="PMDButton" name="birdViewButton">
     <property name="text">
      <string>Bird View</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PMDButton</class>
   <extends>QPushButton</extends>
   <header>PMDButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
